.github/workflows/ci.yml

```diff
@@ -0,0 +1,52 @@
+name: CI
+on: [push, pull_request]
+jobs:
+
+  qmake:
+    name: "qmake on ${{ matrix.runner }}"
+    runs-on: "${{ matrix.runner }}"
+    strategy:
+      matrix:
+        runner:
+          - "ubuntu-16.04"
+          - "ubuntu-18.04"
+          - "ubuntu-20.04"
+    env:
+      DEBIAN_FRONTEND: noninteractive
+    steps:
+      - uses: actions/checkout@v1
```

Comment:
Ah, and you could use checkout@v2 here (and below).

Reply:
Done.